src/pyOpenMS/pxds/IonIdentityMolecularNetworking.pxd

```diff
@@ -0,0 +1,59 @@
+from Types cimport *
+from String cimport *
+from ConsensusMap cimport *
+
+cdef extern from "<OpenMS/ANALYSIS/ID/IonIdentityMolecularNetworking.h>" namespace "OpenMS":
+
+    cdef cppclass IonIdentityMolecularNetworking "OpenMS::IonIdentityMolecularNetworking":
+        # wrap-doc:
+        #  Includes the necessary functions to generate filed required for GNPS ion identity molecular networking (IIMN).
+
+        IonIdentityMolecularNetworking() nogil except +
+
+        # wrap static method:
+        void annotateConsensusMap(ConsensusMap& consensus_map) nogil except +
```

Comment:
Usually static functions are wrapped as free functions and then attached to the class with "wrap-attach".
I will look for an example and get back to you.